deploy.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
For this script can you add a comment with the site you used as the walkthrough in case we need to reference it again later.

Reply:
It's been added.